benchmarks/dynamo/common.py

```diff
@@ -725,11 +727,6 @@ def maybe_mark_profile(*args, **kwargs):
                     collect_outputs=args.collect_outputs,
                 )
 
-            if should_check_result:
```

Comment:
why remove this? it is used by other backends, isn't it?

Reply:
`is_correct ` is actually not being init in this code. when I run it I got an error saying `is_correct ` is being used before assign.

`is_correct` is also not being used any places further down this function.  I suspect this is a dead code.

Reply:
@anijain2305 do you know, are we still using 'speedup_experiment' for CI? do we not set 'should_check_outputs'?  maybe indeed this is dead and we should remove

Reply:
`should_check_result` is set above if `args.randomize_input` is on. I think `randomize_input` is only used manually (not in CI).

Reply:
Let's remove the assignment to should_check_result as well.

My guess is maybe someone remove the definition of is_correct and the handling of is_correct some how. Removing this part looks good to me.

Reply:
done